ocs_ci/ocs/resources/mcg_bucket.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from abc import ABC, abstractmethod
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility import templating
+from ocs_ci.utility.utils import run_mcg_cmd
+from tests.helpers import create_unique_resource_name, create_resource
+
+logger = logging.getLogger(name=__file__)
+
+
+class MCGBucket(ABC):
```

Comment:
Why not inheriting from OCS to get all the methods come with it?

Reply:
Seeing as it's not relevant in the S3 and CLI cases (and it created a clash with the existing `self.name` that can't be refactored easily), and due to the simplicity of the class, I chose to not inherit it. I also don't see many immediate uses to inheriting it - would be glad to have some examples if it's critical, might've missed something :)

Reply:
ack